dart/lib/src/scope.dart

```diff
@@ -157,23 +145,21 @@ class Scope {
 
   /// Sets an extra to the Scope
   void setExtra(String key, dynamic value) {
-    assert(key != null, "Key can't be null");
-    assert(value != null, "Value can't be null");
-
     _extra[key] = value;
   }
 
   /// Removes an extra from the Scope
   void removeExtra(String key) => _extra.remove(key);
 
-  Future<SentryEvent> applyToEvent(SentryEvent event, dynamic hint) async {
+  Future<SentryEvent?> applyToEvent(SentryEvent event, dynamic hint) async {
```

Comment:
```suggestion
  Future<SentryEvent?> applyToEvent(SentryEvent event, dynamic hint?) async {
```

is `dynamic` special case? because every `hint` should be nullable

Reply:
`dynamic` is null by default, even without a `?`. I put that also in the notes of the PR itself

Reply:
ops missed that. thanks